### PR TITLE
[samaritan] New keyboard

### PR DESCRIPTION
**Pull request by LornaSIL**

### CODE REVIEW

File: release/t/takri_inscript/HISTORY.md [hunk 1]
```diff
@@ -3,4 +3,4 @@ Takri - Inscript Change History
 
 1.0 (2024-05-21)
 ----------------
-* Created by 
+* Created by Lorna Evans
```
Comment (darcywong00) [inline release/t/takri_inscript/HISTORY.md:6]: Does this belong on a separate PR?

File: release/h/hanifi_rohingya/HISTORY.md [hunk 2]
```diff
@@ -3,4 +3,4 @@ Hanifi Rohingya Change History
 
 1.0 (2024-05-03)
 ----------------
-* Created by 
+* Created by Lorna Evans
```
Comment (darcywong00) [inline release/h/hanifi_rohingya/HISTORY.md:6]: Does this belong on a separate PR?

File: release/t/takri_inscript/HISTORY.md (hunk 1 above)
Comment (DavidLRowe) [reply release/t/takri_inscript/HISTORY.md:6]: Ideally perhaps Yes, but it seems fine since it doesn't affect anything in the keyboard package.

File: release/h/hanifi_rohingya/HISTORY.md (hunk 2 above)
Comment (DavidLRowe) [reply release/h/hanifi_rohingya/HISTORY.md:6]: See previous comment.

**Review by DavidLRowe — APPROVED** [review]
Image for shift layer has a square box on the 4/$ key. Is this intended? If not, dismiss the review and change.

**Comment by LornaSIL** [comment]
> Image for shift layer has a square box on the 4/$ key. Is this intended?

Not exactly, but I couldn't find a font with the script and that symbol. I think we can let it go for now.
